docs/sources/setup-grafana/configure-security/configure-authentication/saml-ui/index.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Learn how to configure SAML authentication in Grafana's UI.
+menuTitle: Configure SAML authentication in UI
+title: Configure SAML authentication in Grafana's UI
+weight: 1150
+---
+
+# Configure SAML authentication in Grafana
+
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../../introduction/grafana-enterprise/" >}}) version 10.0 and later, and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
+
+You can configure SAML authentication in Grafana either through the UI or the Grafana config file.
+
+Grafana SAML UI has the following advantages over configuring SAML in the Grafana config file:
+
+- It is accessible by hosted Grafana users;
+- It doesn't require Grafana to be restarted after a configuration update;
+- Access to SAML UI only requires access to SAML settings, so it can be used by users with limited access to Grafana's configuration.
+
+Configuration in the UI takes precedence over configuration in the Grafana config file.
+
+Learn more about [SAML authentication in Grafana and how to set it up through the Grafana config file]({{< relref "../saml/" >}}).
+
+# SAML UI
+
+SAML UI is located under **Authentication > Authentication > SAML**.
+
+You can save your SAML configuration, see SAML integration status and enable or disable SAML authentication on the top right-hand side.
+If your SAML configuration is not valid or is incomplete, you will be notified about it when saving the configuration.
```

Comment:
This seems like detailed information that belongs nested within the steps.

Reply:
I'm not sure about this one - the idea is that you can save the configuration at any point, so I don't really know where to list it in the steps (after each section? or only after the final section?)

Reply:
reading you comment again and realizing you're asking if we should have a Save step after the user has completed each tab.

I guess that depends on how likely it might be for the user to lose their information by mistake. So for example, if I complete the first tab and then click somewhere random (like trying to navigate to dashboards or something), am I prompted to Save? If not, then we might think about adding more frequent Save steps.

Reply:
I think currently you're not prompted to save, and it also doesn't autosave, so we should probably add the `save` step a few times. But I'll double check with Alex if this is the case.